tests/parshaping-simple.expected

```diff
@@ -0,0 +1,297 @@
+Mx 	273.7001
+T	68 81 92 w=13.5352	(any)
+Mx 	120.0467
+My 	179.9364
+T	76 86 82 83 72 85 76 80 72 87 85 76 70 w=49.7900	(isoperimetric)
+Mx 	172.6800
+T	83 82 79 92 74 82 81 w=30.0542	(polygon)
+Mx 	202.7342
+T	17 w=2.0610	(.)
+Mx 	207.6386
+T	36 81 71 w=15.0029	(And)
+Mx 	225.4849
+T	73 88 85 87 75 72 85 w=26.9165	(further)
+Mx 	252.4014
+T	15 w=2.0610	(,)
+Mx 	257.3058
+T	82 73 w=7.3257	(of)
+Mx 	267.4749
+T	87 75 72 86 72 w=19.8545	(these)
+Mx 	121.3574
+My 	190.7364
+T	70 76 85 70 88 80 86 70 85 76 69 72 71 w=52.6860	(circumscribed)
+Mx 	178.5729
+T	83 82 79 92 74 82 81 86 w=33.5303	(polygons)
+Mx 	212.1032
+T	15 w=2.0610	(,)
+Mx 	218.6938
+T	87 75 72 w=12.2212	(the)
+Mx 	235.4444
+T	82 81 72 w=13.6494	(one)
+Mx 	253.6234
+T	87 75 68 87 w=15.2930	(that)
+Mx 	273.4458
+T	75 68 86 w=12.5728	(has)
+Mx 	124.1424
+My 	201.5364
+T	87 75 72 w=12.2212	(the)
+Mx 	139.3438
+T	74 85 72 68 87 72 85 w=27.0615	(greater)
+Mx 	169.3855
+T	81 88 80 69 72 85 w=29.2544	(number)
+Mx 	201.6200
+T	82 73 w=7.3257	(of)
+Mx 	211.9259
+T	86 76 71 72 86 w=18.2285	(sides)
+Mx 	233.1346
+T	75 68 86 w=12.5728	(has)
+Mx 	248.6875
+T	68 w=4.1309	(a)
+Mx 	255.7985
+T	86 80 68 79 79 72 85 w=27.4351	(smaller)
+Mx 	128.5655
+My 	212.3364
+T	68 85 72 68 w=15.9829	(area)
+Mx 	146.7946
+T	87 75 68 81 w=17.1606	(than)
+Mx 	166.2015
+T	87 75 72 w=12.2212	(the)
+Mx 	180.6689
+T	82 81 72 w=13.6494	(one)
+Mx 	196.5645
+T	87 75 68 87 w=15.2930	(that)
+Mx 	214.1036
+T	75 68 86 w=12.5728	(has)
+Mx 	228.9226
+T	68 w=4.1309	(a)
+Mx 	235.2996
+T	79 72 86 86 72 85 w=21.2695	(lesser)
+Mx 	258.8154
+T	81 88 80 w=16.9629	(num)
+Mx 	275.7783
+T	16 w=3.0322	(-)
+Mx 	134.9421
+My 	223.1364
+T	69 72 85 w=12.2915	(ber)
+Mx 	147.2336
+T	30 w=2.0610	(;)
+Mx 	152.1687
+T	69 88 87 w=12.4365	(but)
+Mx 	164.6052
+T	15 w=2.0610	(,)
+Mx 	169.5403
+T	82 81 w=9.4922	(on)
+Mx 	181.9065
+T	87 75 72 w=12.2212	(the)
+Mx 	197.0017
+T	82 87 75 72 85 w=20.3115	(other)
+Mx 	220.1873
+T	75 68 81 71 w=18.7427	(hand)
+Mx 	238.9300
+T	15 w=2.0610	(,)
+Mx 	243.8650
+T	87 75 72 w=12.2212	(the)
+Mx 	258.9603
+T	76 86 82 w=10.4414	(iso)
+Mx 	269.4017
+T	16 w=3.0322	(-)
+Mx 	143.8940
+My 	233.9364
+T	83 72 w=8.9253	(pe)
+Mx 	152.8193
+T	85 76 w=6.0029	(ri)
+Mx 	158.8222
+T	80 72 w=11.3862	(me)
+Mx 	170.2084
+T	87 w=3.0981	(t)
+Mx 	173.3066
+T	85 76 70 w=9.9360	(ric)
+Mx 	185.7672
+T	83 82 79 92 74 82 81 w=30.0542	(polygon)
+Mx 	218.3460
+T	87 75 68 87 w=15.2930	(that)
+Mx 	236.1635
+T	75 68 86 w=12.5728	(has)
+Mx 	251.2609
+T	87 75 72 w=12.2212	(the)
+Mx 	156.8758
+My 	244.7364
+T	74 85 72 68 87 72 85 w=27.0615	(greater)
+Mx 	187.8554
+T	81 88 80 69 72 85 w=29.2544	(number)
+Mx 	221.0279
+T	82 73 w=7.3257	(of)
+Mx 	232.2716
+T	86 76 71 72 86 w=18.2285	(sides)
+Mx 	179.7266
+My 	255.5364
+T	76 86 w=5.9150	(is)
+Mx 	188.0200
+T	87 75 72 w=12.2212	(the)
+Mx 	202.6196
+T	79 68 85 74 72 85 w=22.2451	(larger)
+Mx 	224.8647
+T	17 w=2.0610	(.)
+Mx 	201.3418
+My 	565.3031
+Set font 	Gentium Plus;10;400;;normal;;LTR
+T	20 w=4.6924	(1)
```

Comment:
As I was taught by @alerque before you, tests should set `\nofolios` :-)

Reply:
Indeed! Done.